### PR TITLE
Various Spelling / Grammar fixes

### PR DESCRIPTION
**Pull request by trentbroderick**
Fixed a few instances of misspellings and sentences that read incorrectly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -75,19 +75,19 @@ ETL_PATH_PREFIX="/my/cool/path/kubecost/var/config" CONFIG_PATH="/my/cool/path/k
...
 - Temporary workaround: Copy the default.json file in this project at cloud/default.json to /models/default.json on the machine your test is running on. TODO: fix this and inject the cloud/default.json path into provider.go.
 - Navigate to cost-model/test
 - Run `go test -timeout 700s` from the testing directory. The tests right now take about 10 minutes (600s) to run because they bring up and down pods and wait for Prometheus to scrape data about them.
 
 ## Certification of Origin
 
-By contributing to this project you certify that your contribution was created in whole or in part by you and that you have the right to submit it under the open source license indicated in the project. In other words, please confirm that you, as a contributor, have the legal right to make the contribution.
+By contributing to this project, you certify that your contribution was created in whole or in part by you and that you have the right to submit it under the open source license indicated in the project. In other words, please confirm that you, as a contributor, have the legal right to make the contribution.
```
Comment (michaelmdresser) [inline CONTRIBUTING.md:86]: @dwbrown2 is there any legal implication to making this change? Just want to make sure we're careful with language like this.

**Comment by Adam-Stack-PM** [comment]
@michaelmdresser, Dose this need to go in the release, ok to push to v1.96, or is it outside the normal flow?

File: CONTRIBUTING.md (hunk 1 above)
Comment (dwbrown2) [reply CONTRIBUTING.md:86]: I'm no lawyer, but don't think so :)

**Comment by dwbrown2** [comment]
Doesn't need to be in a release branch, but this LGTM

**Review by michaelmdresser — APPROVED** [review]

**Comment by michaelmdresser** [comment]
Outside normal flow, we don't bundle documentation with the release.

**Comment by Adam-Stack-PM** [comment]
@trentbroderick Thanks for this work. Please review any comments and merge when ready.

**Pull request merged**